### PR TITLE
Add command line args

### PR DESCRIPTION
**Pull request by ptbrowne**
Make the children example runnable with arguments so that you can specify a hierarchy dimension and a cube.

Examples

```
yarn example ./examples/children.ts --cube https://environment.ld.admin.ch/foen/fab_Offentliche_Ausgaben_test3/1 list-dimensions
yarn example ./examples/children.ts --cube https://environment.ld.admin.ch/foen/fab_Offentliche_Ausgaben_test3/1 list-values # Does not output children
yarn example ./examples/children.ts --cube https://environment.ld.admin.ch/foen/ubd0104/8 list-values
yarn example ./examples/children.ts --cube https://environment.ld.admin.ch/foen/ubd0104/8 list-dimensions
```

### CODE REVIEW

File: Untitled-1.nnb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "cells": [
+        {
+            "language": "typescript",
+            "source": [
+                "const hierarchies = require('./resources')"
+            ],
+            "outputs": []
+        }
+    ]
+}
```
Comment (tpluscode) [inline Untitled-1.nnb]: What is this file? :)

File: .changeset/cli-args-children-example.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@zazuko/cube-hierarchy-query": patch
+---
+
+Add CLI arguments to "children" example to specify cube and dimension
```
Comment (tpluscode) [inline .changeset/cli-args-children-example.md]: I'm not sure we actually need a changeset. As in, do modifications in the examples warrant publishing a new package version?

File: yarn.lock [hunk 3]
```diff
@@ -3284,6 +3294,24 @@ rdf-normalize@^1.0.0:
...
+    rdf-sink "^1.0.0"
+    readable-stream "^2.2.9"
+
+rdf-sink@^1.0.0:
+  version "1.0.1"
+  resolved "https://registry.yarnpkg.com/rdf-sink/-/rdf-sink-1.0.1.tgz#e82f760233165f1cc7c58e55a1c6345644b31d78"
+  integrity sha512-AytWds7C3HdYOr0nLpLcqvaPnbNFMlZ+u3gccYPzISxBvByMIfMzKbMbCzjQTZVmnVMEfCzACm2+MXeASAACAg==
+  dependencies:
+    rdf-ext "^1.1.0"
+    readable-stream "^2.3.5"
```
Comment (tpluscode) [inline yarn.lock]: Those are some very old libraries. You should use `@rdfjs/parser-n3` and `@rdfjs/sink` instead

File: yarn.lock [hunk 4]
```diff
@@ -3234,6 +3239,11 @@ rdf-data-factory@^1.1.0:
   dependencies:
     "@rdfjs/types" "*"
 
+rdf-data-model@^1.0.0:
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/rdf-data-model/-/rdf-data-model-1.0.0.tgz#e768e1c2cd904c186471b20d1a4fce77c7712a4b"
+  integrity sha512-waBjxCLPB1GeHTBibzsXfr897XwnM+m6gZgmCD5kgRlsq/uarUISRtLjTmRS9BNtU9GCdFMyA0sWQVHv1VzO4g==
```
Comment (tpluscode) [inline yarn.lock]: DItto, please replace with `@rdfjs/data-model`

**Review by tpluscode — CHANGES_REQUESTED** [review]
For now it's a little hard to do a proper review with all the styling changes. 

On top of the comments about dependencies I made, please make sure that your IDE uses the eslint config. It should in fact have kicked-in as a precommit hook. Have you committed with `--no-verify` flag? :)

File: yarn.lock (hunk 4 above)
Comment (ptbrowne) [reply yarn.lock]: Those were not necessary, I removed them, not sure why it was added, thanks !

File: yarn.lock (hunk 3 above)
Comment (ptbrowne) [reply yarn.lock]: Those were not necessary, I removed them, not sure why it was added, thanks !

**Comment by ptbrowne** [comment]
Thanks tpluscode for reviewing, seems like my prettier formatter in vscode was conflicting with eslint. I put a .prettierignore file in the repository to fix the problem, I hope this is OK (I tried to tell vscode to only format with prettier if a config file was present but .editorconfig is detected as a valid config file, hence the extension is enabled).

File: package.json [hunk 5]
```diff
@@ -38,8 +38,9 @@
     "@rdfjs/namespace": "^2",
     "@tpluscode/rdf-ns-builders": "^1.1.0",
     "@tpluscode/sparql-builder": "^0.3.19",
-    "@types/sparql-http-client": "^2.2.8",
     "@types/clownface": "^1.5.0",
+    "@types/sparql-http-client": "^2.2.8",
+    "argparse": "^2.0.1",
```
Comment (tpluscode) [inline package.json]: Being used only in the example, I think this can become a dev dependency

**Review by tpluscode — CHANGES_REQUESTED** [review]
Please remove the last leftovers using the `hierarchy` function
You'll also need update the test code in `resource.test.ts` to something like

```diff
-      const [result] = await query.execute(streamClient, $rdf)
+      const { children: [result] } = await query.execute(streamClient, $rdf)
```

File: package.json (hunk 5 above)
Comment (ptbrowne) [reply package.json]: Yep, thanks !

**Review by tpluscode — APPROVED** [review]

**Pull request merged**